README.md

```diff
@@ -0,0 +1,28 @@
+# GitHub Actions - Add PR to project
+
+Add community-created pull requests to the [Community Contributions project](https://github.com/orgs/silverstripe/projects/4) when they're opened.
+
+## Usage
+
+**.github/workflows/add-pr-to-project.yml**
+```yml
+name: Add pull request to project
+
+on:
+  pull_request:
+    types:
+      - opened
```

Comment:
What happens when a PR is closed before merging? Is it removed from the project board? Closed?

What happens when a PR is merged?

Reply:
If we did need a github action to remove items from the project that'd be a separate action - but in any case they get automatically given the "done" status and are "archived" after that.